js/index.js

```diff
@@ -0,0 +1,81 @@
+import { Professions } from "./enums.js";
+const persons = [{
+        firsName: 'Kirakos',
+        lastName: 'Kirakosyan',
+        age: 30,
+        profession: Professions.HR
+    },
+    {
+        firsName: 'Vazgen',
+        lastName: 'Vazgenyan',
+        age: 18,
+        profession: Professions.Developer
+    },
+    {
+        firsName: 'Hamlet',
+        lastName: 'Hamletyan',
+        age: 25,
+        profession: Professions.Manager
+    },
+    {
+        firsName: 'Grigor',
+        lastName: 'Grigoryan',
+        age: 23,
+        profession: Professions.Marketing
+    },
+    {
+        firsName: 'Vazgen',
+        lastName: 'Hamletyan',
+        age: 16,
+        profession: Professions.QA
+    },
+    {
+        firsName: 'Vazgen',
+        lastName: 'Aperyan',
+        age: 22,
+        profession: Professions.HR
+    },
+    {
+        firsName: 'Vazgen',
+        lastName: 'Karapetkyan',
+        age: 28,
+        profession: Professions.Developer
+    },
+    {
+        firsName: 'Vazgen',
+        lastName: 'Mkoyan',
+        age: 30,
+        profession: Professions.Manager
+    }
+];
+
+function peopleName(name) {
```

Comment:
Please change the function name to more understandable so we will understand what it does.
For example findPersonByName or searchPersonByName